HINWEISE.md

```diff
@@ -47,4 +47,8 @@ $Q=\text{was noch viel tolleres}$
 
 - Die einzelnen Zeilen der Formel werden im Mathemathikmodus geschrieben. Die Erklärung der Formelzeichen auch. Siehe dazu auch das HAWA-Dokument.
 
-- Normale Anführungszeichen ("") entsprechen nicht der Deutschen Rechtschreibung. Hierzu das Kommando `\striche{Text}` verwenden.
\ No newline at end of file
+- Normale Anführungszeichen ("") entsprechen nicht der Deutschen Rechtschreibung. Hierzu das Kommando `\striche{Text}` verwenden.
+
+- Um aus der PDF-Ansicht an die entsprechende Stelle im Code zu gelangen, diese mit gehaltener STRG-Taste anklicken.
```

Comment:
würde hier (und im nächsten punkt) noch explizit schreiben dass das reverse bzw. forward synctex ist

Reply:
naja ich persönlich weiß nicht mal welche Richtung da welche ist. So ist es eindeutig beschrieben. Man könnte das noch in Klammern schreiben (das kannst du gern tun), aber wenn jemand auf die Hinweise geht, keine Ahnung hat und "forward synctex" liest hat der schon kein Bock mehr.